### PR TITLE
Pr/readme

### PR DESCRIPTION
**Pull request by eliatib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,24 @@ Contact Form Plugin
 ## Features
 
 * Extend contact form
+* Add a ReCaptcha Verification to the contact form (only supports invisible ReCaptcha V2 keys)
```
Comment (ondrej-kuhnel) [inline README.md]: remove word `keys`

File: README.md [hunk 2]
```diff
@@ -81,10 +96,16 @@ Contact Form Plugin
...
     ```
+
 7. Create and run doctrine database migrations.
 
 For the guide how to use your own entity see [Sylius docs - Customizing Models](https://docs.sylius.com/en/1.6/customization/model.html)
 
+### Usage
+
+* Parameters can be left empty if you want to run the plugin without recaptcha verification.
+* The plugin is made to work with invisible V2 recaptcha keys,it is essential to select this version during their creation.
```
Comment (ondrej-kuhnel) [inline README.md]: remove word `keys`

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eliatib) [reply README.md]: DONE

File: README.md (hunk 2 above)
Comment (eliatib) [reply README.md]: DONE

**Review by ondrej-kuhnel — APPROVED** [review]

**Pull request merged**